content/guides/mobile/ios/quick-start/0-index.md

```diff
@@ -0,0 +1,34 @@
+---
+type: quick-start
+hide_step_number: true
+hide_in_page_nav: true
+---
+
+# iOS SDK Quick Start
+
+The Box iOS SDK is a [Swift language](https://developer.apple.com/swift/)
+wrapper for adding the Box API into your iOS project.
+
```

Comment:
Maybe add a nice image like we've done with both the Metadata and Postman quick starts?

Reply:
Let me revisit this once I have a few more applications prepped to be able to show something nice.